packages/analytics/src/integrations/Omnitracking/omnitracking-helper.ts

```diff
@@ -533,7 +537,7 @@ export const getCheckoutEventGenericProperties = (
   }
 
   return {
-    orderCode: data.properties?.orderId,
+    orderCode: orderInfo.data.orderCode,
     checkoutOrderId: data.properties?.checkoutOrderId,
```

Comment:
Shouldn't `checkoutOrderId` be also set from the `orderInfo` value you got instead of accessing `checkoutOrderId` directly from `data.properties`?